includes/polling/os/sm-os.inc.php

```diff
@@ -0,0 +1,3 @@
+<?php
+$version = snmp_get($device, '.1.3.6.1.4.1.3373.1103.7.2.0', '-Oqv', '', '');
```

Comment:
As you've got the MIB files, can you use them for these snmp lookups.

Please also convert to snmp_get_multi_oid() instead of two separate calls.

Reply:
@murrant said I do not need to declare the mibs on numeric OID's.

Reply:
But you have the mibs so you should use them. Otherwise, no point in uploading the mibs.